automation/project/Dependencies.scala

```diff
@@ -4,6 +4,8 @@ object Dependencies {
   val jacksonV = "2.8.4"
   val akkaV = "2.4.17"
   val akkaHttpV = "10.0.5"
+  val workbenchGoogleV  = "0.10-d97f551"
```

Comment:
Oops it looks like we both did this:
https://github.com/broadinstitute/leonardo/pull/80